src/domain/event/__tests__/EditEventPage.test.tsx

```diff
@@ -316,7 +314,7 @@ describe('Language selection', () => {
     ```
     NOTE: CreateEventPAge testes this too, but there all the fields are newly filled and not presaved.
   */
-  test('filled fields for unselected languages are cleared when submitting the form', async () => {
+  test.skip('filled fields for unselected languages are cleared when submitting the form', async () => {
```

Comment:
Is this skipped in purpose?

Reply:
Actually this is already skipped in current master:
- https://github.com/City-of-Helsinki/palvelutarjotin-admin/blob/428e0e45acd611c7854c94ed62f0b88552e7ad9a/src/domain/event/__tests__/EditEventPage.test.tsx#L268

I just tried removing the skip, and put it back.